xml/http_wwwauth.xml

```diff
@@ -38,9 +33,13 @@
     <param pos="0" name="os.product" value="IOS"/>
     <param pos="0" name="os.version" value="12"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:cisco:ios:12"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="0" name="hw.device" value="Router"/>
   </fingerprint>
   <fingerprint pattern="^(?:Basic|Digest) realm=.level[ _]\d\d?[ _]access.$">
```

Comment:
Why no quote change here?

Reply:
I should have worded that differently. I had replaced them opportunistically. I've gone back now and replaced them all. Thanks for the feedback.